### ISSUE TITLE
Can you clarify for me what "close all open tabs" means?  Right now, my browser is showing both stashed & unstahsed tabs (a lot of them).  If I choose "close all stashed tabs, does that just close the tabs ikn the browser, or does it also delete those bookmarks from my Stashed Tabs lists?  Thanks.

### ISSUE THREAD

**Issue by hallsvt** [josh-berry/tab-stash#141]
### System Details

- **OS Version:** [e.g. Windows 10, or Ubuntu 18.04]
- **Browser Version:** [e.g. Firefox 60.0.2]
- **Tab Stash Version:** [check the Tab Stash entry in `about:addons`]

**Installed Browser Extensions:**

- 
- 

### Additional Details

<hr>
<em>Readers: If you are also experiencing this bug, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button.  You are welcome to leave comments and discuss the issue if you have more details to add, but "Me too!" comments are not counted by the voting system.</em>

**Comment by josh-berry** [comment]
* **"Close all open tabs"** hides/closes all open tabs (stashed and unstashed) and gets you back to a single new tab in the window. It is a combination of the next two options:
* **"Close all stashed tabs"** will hide/close only those open tabs that are already in your stash. *The bookmarks will be saved, so the tabs are still in your stash and won't be lost.*
  * By default, Tab Stash will hide stashed tabs (instead of closing them), so they're ready for you again later without having to be fully reloaded. But if you have too many tabs open, it will start unloading these hidden tabs so you don't run out of memory. You can control all of this in the settings if you want.
* **"Close all unstashed tabs"** will close all the tabs that are NOT in your stash. Since these tabs are not stashed/saved, they WILL be lost (but may be recoverable through Firefox's "Recently Closed Tabs" menu).

**"Deleting"** groups and tabs means deleting the bookmarks out of your stash entirely. But if you make a mistake and delete something out of your stash that you didn't intend to, you can always go into the "Deleted Items" (from the main tab Stash menu next to the search bar) to get them back.

So in summary, "Closing" gets something out of your tab bar (but doesn't disturb the bookmarks) and "Deleting" removes the bookmarks/stash entries.

Hope this helps!
